pkg/stream/output_multiple.go

```diff
@@ -2,20 +2,36 @@ package stream
 
 import (
 	"context"
-	"fmt"
 	"github.com/applike/gosoline/pkg/cfg"
 	"github.com/applike/gosoline/pkg/mon"
 	"github.com/hashicorp/go-multierror"
 )
 
+type MultipleOutputConfiguration struct {
```

Comment:
this struct belongs into `output_configurable.go` together with the other configuration structs